xcm/src/v0/junction.rs

```diff
@@ -85,21 +120,28 @@ pub enum Junction {
 	///
 	/// Not currently used except as a fallback when deriving ancestry.
 	OnlyChild,
+	/// A pluralistic body existing within consensus.
+	///
+	/// Typical to be used to represent a governance origin of a chain, but could in principle be used to represent
+	/// things such as multisigs also.
+	Plurality { id: BodyId, part: BodyPart },
```

Comment:
what will be the `id` if this is used to represent a multisig or some custom dynamic collective? `BodyId::Named(EncodedMutiSigAddressOrCollectiveId)`?

Reply:
If the multisig has an address then it is probably best to simply use it as a multisig and not expose the plurality aspect. In that case you'd just collapse it to a `AccountId32` or whatever.

This is specifically for the case where you want to expose the nature of a collective over XCM. Right now I don't see much use beyond exposing major governance bodies between chains. If you wanted to expose more, then you'll need to come up with your own naming (or make suggestions for additional entries in the `BodyId` enum).